cdc_rsync_cli/params.cc

```diff
@@ -324,6 +316,35 @@ bool CheckParameters(const Parameters& params, bool help) {
               << std::endl;
   }
 
+  if (params.sources.empty() && params.destination.empty()) {
```

Comment:
It provides a slightly better error message. Otherwise, you could go

cdc_rsync --port=1234
-> Error: Missing destination
cdc_rsync --port=1234 /mnt/developer
-> Error: Missing source